.circleci/config.yml

```diff
@@ -78,10 +78,6 @@ jobs:
           name: Start Test Server
           command: yarn start:test
           background: true
-      - run:
```

Comment:
Were we (supposed to still be) using BrowserStack? Looking though CircleCI logs, this was always failing.

Reply:
Hmm, this one only works in master I believe, because it uses env variables. They are there to make sure that CodeSandbox works eg. in IE11. I think it'd be good to keep it, but only on master. Very strange that the build runs though, wasn't expecting the tests to work... Weird